src/ParaTest/Runners/PHPUnit/Worker.php

```diff
@@ -32,7 +32,12 @@ public function start($wrapperBinary, $token = 1, $uniqueToken = null)
         if ($uniqueToken) {
             $bin .= "UNIQUE_TEST_TOKEN=$uniqueToken ";
         }
-        $bin .= "exec \"$wrapperBinary\"";
+        if (defined('HHVM_VERSION')) {
```

Comment:
Is there another boolean way to see if we're on HHVM than to check `HHVM_VERSION`?